extensions/database-documentation/package.nls.json

```diff
@@ -0,0 +1,5 @@
+{
+	"displayName": "Database Documentation and Diagrams",
+	"description": "Generate Documentation and Diagrams for Database Schemas",
```

Comment:
the description doesn't seem to be accurate.
1. Mention this is only for Microsoft SQL Server and SQL Database.
2. IIUC, this extension works with databases, schemas, tables and views.